### PR TITLE
Added action menu to agent resource access page

### PR DESCRIPTION
**Pull request by norbert989**
# Added action menu to agent resource table rows

- Added action menu with details/view resource and revoke access buttons
- Removed details action from entire row 
- Added details action to resource name only

_NOTE: Removing the details view trigger was necessary as it was causing unexpected results when clicking the action menu. Clicking the action menu would trigger the row click event and open the drawer while ignoring the action menu listener._

# Checklist

- [x] All acceptance criteria are met.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by VirginiaBalseiro — APPROVED** [review]
Clicking on the revoke access button in the details menu causes an error because `accessControl` is undefined

**Review by KyraAssaad — APPROVED** [review]
Yay it looks great! 

Here are a couple of comments - 
1. For some reason, the separator on the sidebar has disappeared. It didn't occur in this ticket because I checked on prod and it's also there. 
Inrupt_PodBrowser
There's not a whole lot of point in fixing it (unless it's super easy) because I want to restyle the whole thing anyway. I'll put this issue in the ticket that I have to restyle the sidebar.

2. I want to make the resource names have underlines now, to make them look more clickable. I'll create a new ticket for that.

3. When I was clicking on different resources on the page, it seems to refresh the profile picture of the agent every time. Don't know if that was introduced here. It's not a huge issue but it kind of "blinks" when you're clicking through things. If it's easily addressable here, that would be great, if not, I can create another ticket for it.

Otherwise, great work! It's so exciting seeing this all come together.

**Comment by KyraAssaad** [comment]
> 1. For some reason, the separator on the sidebar has disappeared. It didn't occur in this ticket because I checked on prod and it's also there.
>    Inrupt_PodBrowser

For anyone aside from me who has eyes, you will see the separator in the screenshot.... it was just my monitor so no problem to address here :)

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
I meant to request changes and accidentally approved: Clicking on the revoke access button in the details menu causes an error because accessControl is undefined

File: components/pages/privacy/resourceAccess/resourceAccessDrawer/index.jsx [hunk 1]
```diff
@@ -256,7 +255,7 @@ export default function ResourceAccessDrawer({
           >
             Remove Access to {resourceName}
           </button>
-          <ConfirmationDialog />
+          {/* <ConfirmationDialog /> */}
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/resourceAccess/resourceAccessDrawer/index.jsx:258]: Why is this commented out as opposed to removed?

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]

File: components/pages/privacy/resourceAccess/resourceAccessDrawer/index.jsx (hunk 1 above)
Comment (norbert989) [reply components/pages/privacy/resourceAccess/resourceAccessDrawer/index.jsx:258]: 🙈  removed it

**Review by VirginiaBalseiro — APPROVED** [review]

**Review by kevin-inrupt — APPROVED** [review]

**Pull request merged**